docs/topics/overall.rst

```diff
@@ -0,0 +1,46 @@
+Overall
+-------
+
+There are things that are assumed to be true on all API requests to solitude.
+And to prevent repetition in the documentation are mentioned here.
+
+Requests
+~~~~~~~~
+
+* It's assumed that all requests are `application/json`. This isn't enforced
+  yet, but could be. If you use curling, all requests are sent with
+  `Content-Type: application/json`.
```

Comment:
They just seem easier to deal with. ``curl -X POST -d foo=bar /baz`` instead of ``curl -H Content-Type:application/json -X POST -d '{"foo":"bar"}' /baz`` -- and for what? It's rare to require nested parameters on input.